docs/execution-providers/OpenVINO-ExecutionProvider.md

```diff
@@ -132,7 +131,7 @@ However, int8 support won't be available for VPU.
 
 Starting from the OpenVINO EP 2021.4 Release, support for external weights is added. OpenVINO™ EP now  supports ONNX models that store weights in external files. It is especially useful for models larger than 2GB because of protobuf limitations.
 
-See the [OpenVINO ONNX Support documentation](https://docs.openvino.ai/latest/openvino_docs_IE_DG_ONNX_Support.html).
+See the [OpenVINO ONNX Support documentation](https://docs.openvino.ai/latest/openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_ONNX.html).
```

Comment:
Please confirm this link @MaajidKhan

Reply:
I couldn't find where the original page went. there's a CN version at https://docs.openvino.ai/cn/latest/openvino_docs_IE_DG_ONNX_Support.html
as you can see the content is kind of different from the page you linked to (using model optimizer, which is probably not what we want to link to for ONNX support) in the context of OnnxRuntime which uses ReadNetwork() to natively read ONNX models.

Reply:
Hello. The links have changed as OV has updated their documentation with their new release. As mentioned earlier, now OV-EP also will use the latest OV 2.0 API's with the upcoming OV-EP release with OV 2022.1.

In OV 2022.1, we don't have ReadNetwork() flow to read an ONNX Model instead we would be having read_model() from the new OV 2.0 API's. Here's the right link: https://docs.openvino.ai/latest/classov_1_1Core.html

Reply:
Thank you!